.github/dependabot.yml

```diff
@@ -140,10 +133,6 @@ updates:
     ignore:
       - dependency-name: "tanzu-framework"
 
-  - package-ecosystem: "gomod"
-    directory: "addons/packages/harbor/2.2.3/test"
-    schedule:
```

Comment:
Rather than removing, this should actually have a schedule value set so it doesn't error on `none`.

Reply:
Hi @stmcginnis , this package is already  defined here https://github.com/vmware-tanzu/community-edition/blob/main/.github/dependabot.yml#L136-L141
Probably a wrong rebase result

Reply:
That one is for harbor 2.3.3, but this one is for 2.2.3. So I think we still want this, just with an actual schedule defined. Looks like maybe a copy/paste problem.

Reply:
Thanks. I didn't notice the version. Let me add the same schedule here